public/modules/articles/articles.viz.js

```diff
@@ -499,6 +515,9 @@ var valign, stacked, articlesScope, aligned = true;
                                         .popover(function () {
                                             return section_hover(d, curS)
                                         })
+                                        .on("mouseenter", function() {
+                                            $(".popover").addClass("articles-popover");
+                                        })
                                         .text(clean_premier(titre_section(sub_section(curS), longlabel)));
                                     ct++;
```

Comment:
_Related follow-up: agreed, the whole d3-bootstrap popovers are so constrained I remember fighting with it to make anything, good job on the hack, but indeed building our own would seem reasonable_